zuul.d/jobs.yaml

```diff
@@ -525,12 +525,13 @@
       - charm-build
     vars:
       tox_extra_args: '-- noble-caracal'
+      charmcraft_channel: '3.x/beta'
       # NOTE: the next two variables need to be updated together, because the
       # constraints have pinning to install the right version of python-libjuju
       # based on what snap channel is being used to bootstrap the controller
       # from.
-      # juju_snap_channel: '3.1/stable'
-      # pip_constraints_url: 'https://raw.githubusercontent.com/openstack-charmers/zaza-openstack-tests/master/constraints/constraints-2024.1.txt'
+      juju_snap_channel: '3.5/stable'
+      pip_constraints_url: 'https://raw.githubusercontent.com/openstack-charmers/zaza-openstack-tests/master/constraints/noble.txt'
```

Comment:
this is the right url: https://raw.githubusercontent.com/openstack-charmers/zaza-openstack-tests/master/constraints/constraints-noble.txt

Reply:
Good catch. Updated in latest commit